pytorch_lightning/trainer/trainer.py

```diff
@@ -112,6 +112,7 @@ def __init__(
         process_position: int = 0,
         num_nodes: int = 1,
         num_processes: int = 1,
+        devices: Optional[Union[List[int], str, int]] = None,
```

Comment:
this will stay None if we set gpus=x, right?
You are not updating the value of devices when gpus=x? And is that intentional?

Reply:
It's intended. It's one way, we are using `devices` as a medium to set `gpus`, `tpu_cores`, etc. Not the other way around.